openapi/atlas-api-transformed.yaml

```diff
@@ -20573,12 +20702,6 @@ components:
     DataLakeTenant:
       type: object
       properties:
-        _privateEndpointHostnames:
```

Comment:
[q] @wtrocki Do properties with underscore have a specific meaning? In this case I see it was changed from `_privateEndpointHostnames` to `privateEndpointHostnames`.

Reply:
This was renamed in the backend - it did not cause SDK-breaking change by coincidence. 
Provided extra context as PM.